moveit_ros/perception/mesh_filter/src/transform_provider.cpp

```diff
@@ -61,7 +55,7 @@ TransformProvider::~TransformProvider()
 void TransformProvider::start()
 {
   stop_ = false;
-  thread_ = thread(&TransformProvider::run, this);
+  thread_ = boost::thread(&TransformProvider::run, this);
```

Comment:
Also, what about using the stl thread?  Is this being used with something external that only takes the boost thread?